techmatters/flex-plugins
# Mask PII identifiers for all channel type contacts / tasks

Primary reviewer: @GPaoloni or @stephenhand 

## Description
- In this PR, the twilio strings and aselo components with PII are masked for all channel types. All Aselo components have been masked. 
- Two twilio related components need work. 
    - TaskListItem.secondLine (Hopefully, there is a better way than using handlebars)
    - Remove IP address in the first message from child (only webchat)
- `maskIdentifiers` flag implemented in this PR will be replaced in the next PR #1017 . 

### Checklist
- [x] New tests added - Tests updated

### Related Issues
Fixes # 1471

### Verification steps
- Change `maskIdentifiers` in all relevant files to `true` and Check for PII where they usually appear in
  - Task List
  - Task panel when a call comes in
  - Task panel during an active call (except the message bubble... still working on this)
  - Info panels for calls
  - Previous calls banner
  - Contact Details 
  - Agents page
